sys/writeFileSyncBrowser.js

```diff
@@ -2,7 +2,14 @@ const { getFile } = require('./files');
 
 const writeFileSync = (path, data, options) => {
   const file = getFile(path);
+  const { translator } = options;
```

Comment:
This will break if options is undefined, so you will want

const writeFileSync = (path, data, options = {}) => {

above